MinStack.java

```diff
@@ -0,0 +1,61 @@
+import java.util.Stack;
+// Time Complexity :  O(1) for all operations 
+
+// Space Complexity : O(N) for storing stack
+// Did this code successfully run on Leetcode : Yes
+// Any problem you faced while coding this :
+
+
+// Your code here along with comments explaining your approach
+/*
+ For getMin operation,we maintain a global variable to store currentMin.
+ This variable will be upidated on every push and pop operation
+*/
+
+public class MinStack {
+
+	int currentMin;
+	Stack<Integer> stack;
+
+	public MinStack() {
+		currentMin = Integer.MAX_VALUE;
+		stack = new Stack<>();
+	}
+
+	public void push(int ele) {
+		if (ele < currentMin) {
```

Comment:
condition is missed 
ele <= currentMin
Input:
["MinStack","push","push","push","getMin","pop","getMin"]
[[],[0],[1],[0],[],[],[]]
Output:
[null,null,null,null,0,null,1]
Expected:
[null,null,null,null,0,null,0]

Reply:
`["MinStack","push","push","push","getMin","pop","getMin"]
[[],[0],[1],[0],[],[],[]]
Output:
[null,null,null,null,0,null,1]
Expected:
[null,null,null,null,0,null,0]`